packages/earthlyls/package.yaml

```
@@ -0,0 +1,34 @@
+---
+name: earthlyls
+description: |
+  A fast language server for earthly.
+homepage: https://github.com/glehmann/earthly
```

I believe the intended URL is:
```suggestion
homepage: https://github.com/glehmann/earthlyls
```